tests/generic_config_updater/test_ecn_config_update.py

```diff
@@ -32,6 +32,10 @@ def ensure_dut_readiness(duthost):
         duthost: DUT host object
     """
     verify_orchagent_running_or_assert(duthost)
+
+    duthost.shell('sonic-db-cli CONFIG_DB hset "WRED_PROFILE|AZURE_LOSSLESS" green_min_threshold 100 \
```

Comment:
Should we remove the config after the test?

Reply:
Yes, thank you.